recipes/python.rb

```diff
@@ -0,0 +1,2 @@
+node.override['python']['pip_location'] = \
+"#{node['python']['prefix_dir']}/bin/pip2.7"
```

Comment:
have you tried setting your interpreter in python-webapp and seeing if that works instead of manually setting the pip version?